emukit/bayesian_optimization/acquisitions/probability_of_improvement.py

```diff
@@ -67,3 +67,56 @@ def evaluate_with_gradients(self, x: np.ndarray) -> Tuple:
     @property
     def has_gradients(self):
         return isinstance(self.model, IDifferentiable)
+    
+    
+    
+class ProbabilityOfFeasibility(Acquisition):
+
+    def __init__(self, model: Union[IModel, IDifferentiable], jitter: np.float64 = np.float64(0)) -> None:
+        """
+        This acquisition computes for a given input point the probability of satisfying the constraint
+        C<0. For more information see:
+
+        Michael A. Gelbart, Jasper Snoek, and Ryan P. Adams,
+        Bayesian Optimization with Unknown Constraints,
+        https://arxiv.org/pdf/1403.5607.pdf
+
+        :param model: The underlying model that provides the predictive mean and variance for the given test points
+        :param jitter: Jitter to balance exploration / exploitation
+        """
+        self.model = model
+        self.jitter = jitter
+
+    def evaluate(self, x: np.ndarray) -> np.ndarray:
+        """
+        Computes the probability of improving over the current best
+
+        :param x: points where the acquisition is evaluated.
+        """
+        mean, variance = self.model.predict(x)
+        standard_deviation = np.sqrt(variance)
+        _, cdf, _ = get_quantiles(self.jitter, 0, mean, standard_deviation)
+
+        return cdf
+
+    def evaluate_with_gradients(self, x: np.ndarray) -> Tuple:
+        """
+        Computes the  probability of improving over the current best and its derivative
```

Comment:
same here, docstring might need to be updated to fit the probability of feasibility acquisition.